frontend/src/components/Header/Header.jsx

```diff
@@ -39,10 +39,11 @@ const Header = () => {
   const isSmall = useMediaQuery("(max-width: 1000px)");
   const [showDrawer, setShowDrawer] = useState(false);
   const [current, setCurrent] = useState(getCurrentPath());
+  const url = window.location.pathname;
```

Comment:
I think you can also use the 'useLocation' hook to get the pathname aswell. Though this is just an FYI since it's basically the same functionality anyways :)